Hunting Queries/AzureActivity/Creating_Anomalous_Number_Of_Resources.txt

```diff
@@ -7,7 +7,7 @@
 //
 // DataSource: #AzureActivity
 //
-// Tactics: #Execution
+// Tactics: #Execution #Impact
```

Comment:
consistency - should there be a comma here?

Reply:
updated with comma